src/utils/method-helpers.js

```diff
@@ -28,3 +29,6 @@ const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
 
 export const delay = (delayed, ms) =>
   Promise.all([delayed, sleep(ms)]).then(([data]) => data);
+
+export const translate = (key, lang, interpolation = null) =>
```

Comment:
Hmm, I'm not exactly sure, but maybe it will help somehow, like an example https://www.npmjs.com/package/react-localize-redux